src/lsdb/core/search/pixel_search.py

```diff
@@ -18,9 +19,30 @@ class PixelSearch(AbstractSearch):
     Does not filter points inside those partitions.
     """
 
-    def __init__(self, pixels: list[tuple[int, int]]):
+    def __init__(self, pixels: tuple[int, int] | HealpixPixel | list[tuple[int, int] | HealpixPixel]):
         super().__init__(fine=False)
-        self.pixels = [HealpixPixel(o, p) for o, p in set(pixels)]
+        if isinstance(pixels, tuple):
```

Comment:
```suggestion
        if isinstance(pixels, tuple) and len(pixels) == 2:
```

Reply:
`get_healpix_pixel` performs this check, so changing the below line takes care of it.